src/app-exception-filters/main-exception.filter.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpException,
+} from '@nestjs/common';
+import { Response } from 'express';
+import { CustomLogger } from 'src/custom-logger/CustomLogger';
+
+@Catch()
+export class MainExceptionFilter implements ExceptionFilter {
+  private readonly logger = new CustomLogger();
```

Comment:
https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging - tutaj jest przyklad jak to wyglada - wy nic wiecej nie potrzebujecie

Reply:
Tak, widziałem to :) Podstawowe użycie Loggera :)

Reply:
poprawione wg wskazówek